tests/rt/prepare_migration.pm

```diff
@@ -0,0 +1,31 @@
+# SUSE's openQA tests
+#
+# Copyright © 2022 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+#
+# Summary: Prepare RT system for migration
+#            - do SCC cleanup
+#            - register RT system
```

Comment:
I would rather have the reasoning which you described in the morning. For instance why do we need the cleanup. The module is small anyway and it is obvious what its steps are. But it is not clear why. wdyt?